recipes/mutyper/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "mutyper" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 28712852efd635440698ea632ecc4252a0023966d2e428953f08498e889f04e3
+
+build:
+  run_exports:
+    - {{ pin_subpackage('mutyper', max_pin="1") }}
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
```

Comment:
python, pip and setuptools should be `host` dependencies

Reply:
@martin-g @mencian thanks both for the suggestions and edits!